python/ray/util/placement_group.py

```diff
@@ -244,14 +244,25 @@ def remove_placement_group(placement_group: PlacementGroup) -> None:
 
 @PublicAPI
 @client_mode_wrap
-def get_placement_group(placement_group_name: str) -> PlacementGroup:
-    """Get a placement group object with a global name.
+def get_placement_group(
+    placement_group_name: str = None, placement_group_id: str = None
+) -> PlacementGroup:
+    """Get a placement group object with a global name / id.
+
+    one of them (name / id) should be given; if both are given, name will be used.
 
     Returns:
-        None if can't find a placement group with the given name.
+        None if can't find a placement group with the given name / id.
         The placement group object otherwise.
     """
     if not placement_group_name:
+        if placement_group_id:
```

Comment:
I think we want to run 

```
    worker = ray._private.worker.global_worker
    worker.check_connected()
```
for the id case as well? Can you add if/else based on a given input in line 269? 

Also we need to add an unit test